lib/chitragupta/chitragupta.js

```diff
@@ -0,0 +1,57 @@
+const { v4: uuidv4 } = require('uuid');
+
+const CLS = require('./cls');
+const UTIL = require('./util');
+const CONSTANTS = require('./constants');
+
+function getUniqueLogId() {
+  return uuidv4();
+}
+
+function setupInitialServerData(request, response, userId) {
+  request.request_id = getUniqueLogId();
+  CLS.set('server', true);
+  CLS.set('request_start_time', new Date());
```

Comment:
let's make sure this is UTC?
https://stackoverflow.com/a/37625198

Reply:
It will always give based on the machine's timezone, no? Should not be required to explicitly generate UTC.

Reply:
Local times are usually a pitfall while testing locally, let's make sure this isn't one :P

Reply:
It’s rather better to have local timezone as it becomes easier for the developer to check the logs, if in UTC, he will spend his time converting that manually to check the correct log